akka-docs/src/test/java/jdocs/remoting/RemoteDeploymentDocTest.java

```diff
@@ -57,7 +59,7 @@ public void demonstrateDeployment() {
         system.actorOf(
             Props.create(SampleActor.class).withDeploy(new Deploy(new RemoteScope(addr))));
     // #deploy
-    assert ref.path().address().equals(addr);
+    assertEquals(ref.path().address(), addr);
```

Comment:
the order for `assertEquals` is `expected, actual` - so perhaps they should be the other way around?

Reply:
yes, i have updated it